src/apps/VirtualApp/VirtualClient.js

```diff
@@ -401,6 +400,28 @@ class VirtualClient extends Component {
     }, 1000);
   };
 
+  myFunction = () => {
```

Comment:
Maybe add a comment marking this sentry debug functionality. 
Maybe rename the methods.